OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -36,6 +36,13 @@ public class CashTricklerInfo : PausableConditionalTraitInfo
 		[Desc("Use resource storage for cash granted.")]
 		public readonly bool UseResourceStorage = false;
 
+		void IRulesetLoaded<ActorInfo>.RulesetLoaded(Ruleset rules, ActorInfo info)
+		{
+			var occupiesSpace = info.TraitInfoOrDefault<IOccupySpaceInfo>();
+			if (occupiesSpace == null && ShowTicks)
```

Comment:
We're not using `occupiesSpace` directly, so why not `!info.HasTraitInfo<IOccupySpaceInfo>()`?